managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,339 @@
+
+If you need to delete a credential, select the credential from the list of credentials. Then, click **Configuration** > **Remove selected Credentials from Inventory**.
+
+#### Credential Types
+
+Each credential type that is used by {{ site.data.product.title_short }} for embedded workflows is detailed in the following sections.
+
+##### SCM
+
+SCM (source control Management) credentials are used with Projects to clone and update local source code Git repositories. Source Control credentials have several attributes that can be configured:
+
+- **Username**: The username to use in conjunction with the source control system.
+
+- **Password**: The password to use in conjunction with the source control system.
+
+- **Private key passphrase**: If the SSH private key used is protected by a passphrase, you can configure a key passphrase for the private key.
+
+- **Private Key**: Copy or drag-and-drop the actual SSH Private Key to be used to authenticate the user to the source control system by using SSH.
+
+### Authoring a workflow
+
+You can configure workflows for use in the following areas during service catalog creation:
+
+- **Dynamic dialog fields** - You (or your users) can attach workflows to a field so that the field becomes populated or refreshed with the results from the running of the workflow.
+- **Service Entry Points** - You (or your users) can set and use workflows for provisioning, reconfiguring, and retiring services.
+
+The Automate model provides flexibility to not only change parts of the provisioning process, but also to allow you to automate other operational tasks. You can create and use your own embedded workflows as needed to automate your operational tasks.
+
+Workflows must be authored in Amazon State Languages (ASL) format. As part of authoring a workflow, you (or your users) must build and support the containers that are run during each step of an embedded workflow.
+
+1. Define the code for the workflow. If your workflow requires the use of any credentials or parameters to be specified, ensure that they are passed in the code.
+
+   The workflow code must be in the Amazon States Language (ASL) format and follow its supported specifications. For more information about Amazon States Language and its specification, see [Amazon States Language Guide](https://states-language.net/).
+
+   Within the workflow code, you need to specify the states that your workflow requires, including any next steps. For `Task` type steps in the workflow, a docker container is called. The container defines what happens for that Task state. For example, a docker container can run to clone a template.
+
+2. Build the docker containers that are required for the workflow.
+
+<!--
+##### Add docker images to a podified deployment and appliances (workflow registries)
+Include:
+* Building docker containers (reference Docker documentation)
+* Push to a registry
+* Add the registry to the appliance/kubernetes system.
+
+-->
```

Comment:
```
Once you have the code for your task resource written you need to bundle it into a docker image.  You can do this by creating a standard Dockerfile (https://docs.docker.com/engine/reference/builder/) and building the image (https://docs.docker.com/engine/reference/commandline/build/).  Then you can push the image to a registry (https://docs.docker.com/engine/reference/commandline/push/) which makes them available to be used by {{ site.data.product.title_short }}.

Once you have pushed your images to an image registry you can add the registry to {{ site.data.product.title_short }}.  On appliances `podman` is used to execute the containers so you would `podman login` (https://docs.podman.io/en/stable/markdown/podman-login.1.html) as the `manageiq` user.
# su manageiq
$ podman login docker.io
Username: 
Password:
Login Succeeded!

It is recommended that you use a docker.io access token (https://docs.docker.com/security/for-developers/access-tokens/) so that the token doesn't expire.
```

TODO kubernetes pull secrets for podified

Reply:
I have added this content.